docs/cloudlinux-os-plus/README.md

```diff
@@ -1467,17 +1481,12 @@ There are the following requirements to use AccelerateWP:
 ### Administrator interface
 
 #### Overview
-In the _CloudLinux Manager → AccelerateWP_ tab an administrator has the opportunity to provide end-users with a suite of features, which on its turn could be activated by end-users.
-
-![](/images/AWPAdmin.png)
-
-By toggling the `Enable AccelerateWP for all users` an administrator provides end-users with AccelerateWP feature.
-Once the feature suite is enabled by administrator, end-users will see an AccelerateWP tab in their cPanel interface and be able to activate the optimization feature.
+In the _CloudLinux Manager → AccelerateWP_ tab an administrator provides the information about active features and users.
 
 #### Suites usage statistics
-Once at least one feature suite is enabled, the AccelerateWP usage statistics is shown.
+When AccelerateWP is enabled, the AccelerateWP usage statistics is shown.
 
-![](/images/AWPStats.png)
+![](./images/awp/AWPStats.png)
```

Comment:
The file `./images/awp/AWPStats.png` has the same contents as the `./images/awp/AWPStatsPremium.png`. Maybe on the first image, there should be no AWP Premium statistics?

Reply:
Done.,